ccpp/config/ccpp_prebuild_config.py

```diff
@@ -114,6 +114,8 @@
     'FV3/ccpp/physics/physics/module_bfmicrophysics.f',
     'FV3/ccpp/physics/physics/multi_gases.F90',
     'FV3/ccpp/physics/physics/module_gfdl_cloud_microphys.F90',
+    'FV3/ccpp/physics/physics/module_mp_nssl_2mom.F90',
+    'FV3/ccpp/physics/physics/mp_nssl_2mom.F90',
```

Comment:
If this file (mp_nssl_2mom.F90) is the CCPP entry point scheme, it should not need to appear in the dependencies list (only the schemes list below).

Reply:
Okay, I think I have this correct now.